recipes/relocate2/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: relocate2
+  version: 2.0.0
+
+source:
+  git_url: https://github.com/stajichlab/RelocaTE2.git
+  git_tag: v2.0.0
+
+requirements:
+  build:
+    - samtools >=1.3.1
+    - bowtie2 >=2.2.8
+    - bedtools >=2.25.0
+    - bwa >=0.6.2
+    - seqtk >=1.2
+    - pysam >=0.9.1
+    - blat
+  run:
+    - samtools 
+    - bowtie2
+    - bedtools
+    - bwa
+    - seqtk
+    - pysam
+    - blat
+
+#test:
+#  commands:
```

Comment:
I have problem when building the package offline. The testing failed and says "run_test.sh: line 3: relocaTE2.py: command not found".  I am not sure I understand how the testing works. But it seems building and installation are fine. Can you give me some suggestion how I should modify the testing?
Thanks.

Jinfeng